src/libraries/Microsoft.PowerFx.Connectors/Public/BaseRuntimeConnectorContext.cs

```diff
@@ -17,11 +17,13 @@ public abstract class BaseRuntimeConnectorContext
 
         internal virtual bool ReturnRawResults { get; } = false;
 
+        internal virtual MediaKind MediaKind { get; set; } = MediaKind.NotBinary;
```

Comment:
>virtual MediaKind MediaKind  [](http://example.com/codeflow?start=17&length=28)

What exactly does this flag mean?

1. Is it global for all connectors?
2. when does the host set it to the non-default value?

Reply:
this is x-ms-media-kind extension which defines if a function returns a byte array if this is a video, an image...
it's per function and defined in the swagger file